lib/cc/engine/rubocop.rb

```diff
@@ -45,7 +45,10 @@ def exclude?(local_path)
       def rubocop_config_store
         @rubocop_config_store ||= begin
           config_store = RuboCop::ConfigStore.new
-          if (config_file = @engine_config["config"])
+
+          if (default_rubocop_config = Dir.glob("/code/.rubocop.yml").first)
```

Comment:
You are 100% correct, I've just checked and this modification is not necessary. I will revert it in a next commit.